### PR TITLE
gh-completion: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #4701

### CODE REVIEW

File: pages/common/gh-completion.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display the subcommand help:
+
+`gh completion`
+
+- Print a completion script to `stdout`:
+
+`gh completion --shell {{bash|zsh|fish|powershell}}`
+
+- Append the `gh` completion script to the `.bashrc`:
```
Comment (bl-ue) [inline pages/common/gh-completion.md:14]:
```suggestion
- Append the `gh` completion script to `~/.bashrc`:
```

File: pages/common/gh-completion.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`gh completion --shell {{bash}} >> {{~/.bashrc}}`
+
+- Append the `gh` completion script to the `.zshrc`:
```
Comment (bl-ue) [inline pages/common/gh-completion.md:18]:
```suggestion
- Append the `gh` completion script to `~/.zshrc`:
```

File: pages/common/gh-completion.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# gh completion
+
+> Generate shell completion scripts for GitHub CLI commands.
+> More information: <https://cli.github.com/manual/gh_completion>.
+
+- Display the subcommand help:
+
+`gh completion`
+
+- Print a completion script to `stdout`:
```
Comment (bl-ue) [inline pages/common/gh-completion.md:10]: This can be simplified, e.g.:
```suggestion
- Print a completion script:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]
Looks good!

By the way, thank you so much for pushing the `gh` page suite to completion!!! 💙

**Review by owenvoke — APPROVED** [review]

**Pull request merged**